content/docs/0.6.0/installation/setup-keptn/index.md

```diff
@@ -21,7 +21,7 @@ Select one of the following options:
   - [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 
 2. Create AKS cluster
-  - Master version >= `1.12.x` (tested version: `1.12.8`)
+  - Master version >= `1.13.x` (tested version: `1.14.8`)
```

Comment:
Did it work with 1.14.8? I thought we need at least 1.15.5?

Reply:
So far it is working.

Reply:
Today I'm having trouble with 1.14.8. I will now test 1.15.5.